environmentmodel/environment-model_ontology.ttl

```diff
@@ -1,17 +1,16 @@
 @prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix environment-model: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environment-model/> .
+@prefix environmentmodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environmentmodel/> .
```

Comment:
it is allowed to have a '-' in the prefix. I would prefer `environment-model` :)
Already documented in the Readme in the modelling guidelines.

Reply:
I totally agree on that:) 
It should be changed now.